datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -8,9 +8,14 @@
 from six import PY3
 from six.moves.urllib.parse import urlparse
 
+if PY3:
+    text_type = str
+else:
+    text_type = unicode  # noqa: F821
+
 
 def ensure_bytes(s):
-    if not isinstance(s, bytes):
+    if isinstance(s, text_type):
```

Comment:
Actually, just reverting won’t work, since this will fail when using `to_string()->ensure_bytes()`: https://github.com/DataDog/integrations-core/blob/28c6182e45d84b63253fd62a96b4f18cb88a896a/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L635

For this PR, I used the same strategy used here: https://github.com/pypa/wheel/blob/master/wheel/util.py

In this case `ensure_bytes` will let pass everything that is **not** `text_type`, but that’s consistent with `ensure_unicode` which less path everything that is not `bytes`.